src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -1217,5 +1237,26 @@ public static void TestingStringsInvalidConversionToGuid(string testString)
 
             JsonTestHelper.AssertThrows<FormatException>(json, (jsonReader) => jsonReader.GetGuid());
         }
+
+        [Theory]
+        [MemberData(nameof(JsonGuidTestData.InvalidGuidTests), MemberType = typeof(JsonGuidTestData))]
+        public static void TryGetGuid_HasValueSequence_False(string testString)
+        {
+            if (string.IsNullOrEmpty(testString))
+                return;
```

Comment:
Why early return?  Nothing in the rest of the body looks like it would fail with either `null` or `""`.